io.mrarm.mcpelauncher-client.json

```diff
@@ -0,0 +1,113 @@
+{
+    "id": "io.mrarm.mcpelauncher-client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "mcpelauncher-client",
+    "build-options" : {
+        "cflags": "-O2 -g -DNDEBUG",
+        "cxxflags": "-O2 -g -DNDEBUG",
+        "config-opts": [ "-DGAMEWINDOW_SYSTEM=GLFW", "-DCMAKE_THREAD_LIBS_INIT=-lpthread", "-DCMAKE_HAVE_THREADS_LIBRARY=1", "-DCMAKE_USE_PTHREADS_INIT=1" ]
```

Comment:
This should be defined under specific module.

Reply:
Ok

Reply:
Done